monitoring/rid_qualifier/test_executor.py

```diff
@@ -1,16 +1,17 @@
 import json
 import uuid
-from typing import List
+from typing import List, Optional
 from monitoring.rid_qualifier.aircraft_state_replayer import TestHarness, TestBuilder
 from monitoring.rid_qualifier.utils import RIDQualifierTestConfiguration, InjectedFlight
 from monitoring.rid_qualifier import display_data_evaluator, reports
 from monitoring.monitorlib.infrastructure import DSSTestSession
 from monitoring.monitorlib.auth import make_auth_adapter
 
 
-def main(test_configuration: RIDQualifierTestConfiguration, auth_spec: str):
-    # This is the configuration for the test.
-    my_test_builder = TestBuilder(test_configuration = test_configuration)
+def main(
+    test_configuration: RIDQualifierTestConfiguration,
```

Comment:
nit: Same as above: when splitting lines, double-indent line continuations to differentiate the single split line from the following lines of code.  Or, indent to match an open paren -- both of the following are ok:

```python
def example(
        p1: Parameter,
        p2: Parameter):
    pass
````

```python
def example(p1: Parameter,
            p2: Parameter):
    pass
````

We want to avoid:

```python
def example(
    p1: Parameter,
    p2: Parameter):
    pass
````

...because it's hard to see the difference between `p1` & `p2` and `pass`